[IMP] recruitment: job positions new images and options (#7949)
odoo/documentation: content/applications/hr/recruitment/new_job.rst

Job positions
=============

> In Odoo *Recruitment*, all job positions are shown on the default :menuselection:`Recruitment`

Author: FYI I went with "main *Recruitment* app dashboard - so no gui on Recruitment for the name, just calling out it's the app's dashboard.

> positions being actively recruited for as well as inactive positions.

Author: i love it!

Job positions
=============

In Odoo *Recruitment*, all job positions are shown on the default dashboard in the *Recruitment*
app. This includes positions that are being actively recruited for, as well as inactive positions.

Each job position is shown in an individual Kanban card. If the job position is active, and
candidates can apply, a :guilabel:`Published` banner appears in the top-right corner of the card.

View submitted applications by clicking anywhere on a job position card.

.. image:: new_job/jobs.png
   :align: center
   :alt: Main dashboard view of Recruitment app showing all job positions.

Create a new job position
=========================

To create a new job position from the main dashboard in the *Recruitment* app, click the

> :guilabel:`New` button in the top-left corner, and a :guilabel:`Create a Job Position` pop-up window

Reviewer: ```suggestion
:guilabel:`New` button in the top-left corner, and a :guilabel:`Create a Job Position` modal
```

   :align: center
   :alt: Edit the job position card.

All the basic information about the job position is listed under the :guilabel:`Recruitment` tab.

> 

Author: the image beneath the description of the various data items to enter is the image for the form- it is the new_job/recruitment-tab.png. I thought having it after the explanations of the fields was helpful since they would read the items, then see what it looks like populated on the form. I could move it up but I thought it made sense there.

  the website.
- :guilabel:`Job Location`: select the physical address for the job. If the job position is remote,
  leave this field blank. This is visible on the website.
- :guilabel:`Email Alias`: enter an email address to which applicants can send a resumé. Once
  emailed, Odoo automatically creates an application for them.
- :guilabel:`Employment Type`: select what type of position the job is, using the drop-down menu.
  The default options are :guilabel:`Permanent`, :guilabel:`Temporary`, :guilabel:`Seasonal`,
  :guilabel:`Interim`, :guilabel:`Full-Time`, and :guilabel:`Part-Time`. This is visible on the
  website.
- :guilabel:`Company`: select the company the job is for. This field only appears if using a
  multi-company database.
- :guilabel:`Target`: enter the number of employees to be hired for this position.
- :guilabel:`Is Published`: activate this option to publish the job online.
- :guilabel:`Website`: select the website the job is published on.
- :guilabel:`Recruiter`: select the person responsible for recruiting this role.
- :guilabel:`Interviewers`: select who should perform the interviews. Multiple people can be
  selected.

> - :guilabel:`Interview Form`: select an :ref:`Interview form <interview>` that applicants fill out
>   prior to their interview.

Reviewer: Please update the custom anchor to include the app/module name :)
```suggestion
- :guilabel:`Interview Form`: select an :ref:`Interview form <recruitment/interview>` that
  applicants fill out prior to their interview.
```

Author: I forgot about adding that first part! The funny thing is, it worked in the 'make fast' when I test it. I assumed if it did- then it was ok. Whoops =)

   selected from a drop-down menu. The text is displayed on the website exactly as it appears in
   this tab.

Finally, enter the job description in the :guilabel:`Job Summary` tab.

.. image:: new_job/recruitment-tab.png
   :align: center

>    :alt: Enter job information details in the recruitment tab.

Reviewer: ```suggestion
   :alt: Enter job information details in the recruitment tab.
   
   .. _recruitment/interview:
```

forms can be as specific or general as desired, and can take the form of a certification, an exam,
or a general questionnaire. Interview forms are determined by the recruitment team.

Before creating an interview form, ensure the proper settings are enabled. Navigate to
:menuselection:`Recruitment app --> Configuration --> Settings`, and under the
:guilabel:`Recruitment Process` section, ensure the :guilabel:`Send Interview Survey` option is
enabled.

> Since there are no pre-configured forms in Odoo, all interview forms must be created. To create an
> interview form, start from the :guilabel:`Recruitment` tab of the :guilabel:`Job Position` form. In
> the :guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
> several options populate beneath the entry: :guilabel:`Create (interview form name)`,
> :guilabel:`Search More...`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`
> and a :guilabel:`Create Interview Form` pop-up window appears.

Reviewer 2: ```suggestion
Since there are no pre-configured forms in Odoo, all interview forms must be created. 

To create an interview form, start from the :guilabel:`Recruitment` tab of the :guilabel:`Job Position` form. In
the :guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
several options populate beneath the entry: :guilabel:`Create (interview form name)`,
:guilabel:`Search More...`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`
and a :guilabel:`Create Interview Form` pop-up window appears.
```

Reviewer 2: an image of the pop-up would be nice here

.. note::
   The option :guilabel:`Search More...` only appears if there are any interview forms already
   created. If no interview forms exist, the only options available are :guilabel:`Create (interview
   form name)`, and :guilabel:`Create and edit...`.

> First, enter a name for the form in the :guilabel:`Name` field. This should be indicative of when

Reviewer 2: From here on down, i think you can shorten this doc a great deal by simply linking to the doc on creating surveys. I just rewrote it from scratch (it hasn't been published yet) and it covers every single element that you mentioned -- along with specific screenshots and elaborated descriptions.

So, you can simply allude to that (w/a link) with something like "...and then proceed to fill out this pop-up from the same way you would a typical survey..."

Author: OMG! Ok that does save about 445 lines or so 0_0

I assume it's the "survey essentials" document?

forms can be as specific or general as desired, and can take the form of a certification, an exam,
or a general questionnaire. Interview forms are determined by the recruitment team.

Before creating an interview form, ensure the proper settings are enabled. Navigate to
:menuselection:`Recruitment app --> Configuration --> Settings`, and under the
:guilabel:`Recruitment Process` section, ensure the :guilabel:`Send Interview Survey` option is
enabled.

Since there are no pre-configured forms in Odoo, all interview forms must be created. To create an
interview form, start from the :guilabel:`Recruitment` tab of the :guilabel:`Job Position` form. In
the :guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
several options populate beneath the entry: :guilabel:`Create (interview form name)`,
:guilabel:`Search More...`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`

> and a :guilabel:`Create Interview Form` pop-up window appears.

Reviewer: ```suggestion
and a :guilabel:`Create Interview Form` modal appears.
```

.. image:: new_job/blank-interview-form.png
   :align: center

>    :alt: The blank interview form that pops-up.

Reviewer: ```suggestion
   :alt: The blank interview form modal.
```

.. note::
   The option :guilabel:`Search More...` only appears if there are any interview forms already
   created. If no interview forms exist, the only options available are :guilabel:`Create (interview
   form name)`, and :guilabel:`Create and edit...`.

> Proceed to fill out the pop-up interview form as a typical survey. For specific directions on how to

Reviewer: ```suggestion
Proceed to fill out the modal interview form as a typical survey. For specific directions on how to
```

      :guilabel:`Choices` section, while the rows are configured in the :guilabel:`Rows` section.

      The method for populating both sections is the same. In the :guilabel:`Answers` tab, click
      :guilabel:`Add a line` in either the :guilabel:`Choices` or :guilabel:`Row` section. A line
      appears, and an answer can be entered. After typing in the answer, click off the line, or

>       press enter to lock in the new answer on the form and have another answer line appear. Repeat

Author: I agree- especially since it's the only time I mention a keyboard key and not a clickable button.

- :guilabel:`Display Progress as`: choose how to display the percentage of questions remaining to
  be answered. This option only appears if either :guilabel:`One page per question` or
  :guilabel:`One page per section` is selected for :guilabel:`Pagination`. The options are:

  - :guilabel:`Percentage left`: display the remaining amount in a percentage (%).
  - :guilabel:`Number`: display the remaining amount in a numerical value.

- :guilabel:`Question Selection`: choose which questions are presented to the applicant. The options
  are:

  - :guilabel:`All questions`: display the entire form, with all questions from all sections.
  - :guilabel:`Randomized per Section`: display only a random selection of questions from each
    section.

- :guilabel:`Allow Roaming`: activate this option if the applicant is able to click a back button to
  go back to previous questions.

.. note::
   Although it is a selectable option, the :guilabel:`Randomized per section` option should only be

>    used if receiving partial information/an incomplete survey from every applicant is acceptable to
>    the business.

Reviewer 3: ```suggestion
   used if receiving partial information/an incomplete survey from every applicant is acceptable.
```
Optional suggestion to simplify wording

Participants
************

This section determines who can access the interview form, and what kinds of restrictions are
imposed.

- :guilabel:`Access Mode`: specify who can access the interview form. The available options are
  either :guilabel:`Anyone with the link` or :guilabel:`Invited people only`.
- :guilabel:`Require Login`: activate this option to require candidates to log in before accessing
  the interview form, whether they have a valid token or not.
- :guilabel:`Limit Attempts`: if there is a limit to how many times someone can attempt to login to
  the interview form, activate this box. When activated, an :guilabel:`attempts` field appears.
  Enter the maximum number of login attempts in the field.

Time & scoring
**************

This section determines how long candidates have to complete an interview form, and how the form is
scored, if at all.

- :guilabel:`Survey Time Limit`: activate this option to limit the time allowed to complete the
  interview form. When selected, a :guilabel:`minutes` field appears next to it. Enter the allotted

>   time for the interview form (using an XX:XX minute/second format) in the field.

Reviewer 3: ```suggestion
  time for the interview form in the field, using an XX:XX minute/second format.
```
Optional to get rid of parentheses

- :guilabel:`Scoring`: choose how the interview form should be scored. The options are:

  - :guilabel:`No scoring`: select this option to not score the form.
  - :guilabel:`Scoring with answers after each page`: select this option to score the interview form
    and display the correct answers for the candidate as they finish each page.
  - :guilabel:`Scoring with answers at the end`: select this option to score the interview form and
    display the correct answers for the candidate after they have completed the interview form.
  - :guilabel:`Scoring without answers`: select this option to score the interview form but not
    display the answers to the candidate.

- :guilabel:`Required Score (%)`: this option appears if one of the scoring options was selected.

>   Enter the percentage the candidate needs to pass the interview form (example, 80.00%). The entry

Reviewer 3: ```suggestion
  Enter the percentage the candidate needs to pass the interview form (e.g., 80.00%). The entry
```
Optional!

>   should be written in an “XX.XX” format.

Reviewer 3: ```suggestion
  should be written in an `XX.XX` format.
```
What do you think of this, since it's something we have to enter ourselves?

Once the interview form has been configured, there are several options available: :ref:`Share
<share>`, :ref:`Create Live Session <live-session>`, :ref:`Test <test>`, and :ref:`Print <print>`

.. _share:

Share
*****

The interview form can be shared with applicants. Typically, this is done directly from the
applicant form, but the survey can also be shared in this view.

To send the form to a candidate, click the :guilabel:`Share` button in the top-left corner of the
survey form. A :guilabel:`Share a Survey` pop-up window appears.

First, select the :guilabel:`Recipients` using the drop-down menu. Multiple selections can be made
in this field. The :guilabel:`Subject` field auto-populates with `Participate to (name of survey)`
but can be modified, if desired.

If any attachments need to be sent with the email, click the :guilabel:`Attachments` button. A file
explorer pop-up window appears. Navigate to the file to add, then click :guilabel:`Open` to select
it and attach it to the email.

If there is a deadline for the survey to be completed, enter the date in the :guilabel:`Answer
deadline` field. Click the field to the right of :guilabel:`Answer deadline` and a calendar pop-up

> window appears. Navigate to the desired date and click on the day to select it. Next, select the
> time using the two drop-down fields, one for the hours, one for the minutes. Finally, click
> :guilabel:`Apply` to select both the date and time.

Reviewer 3: ```suggestion
window appears. Navigate to the desired date and click on the day to select it.

Next, select the time using the two drop-down fields: one for the hours, and one for the minutes. Finally, click
:guilabel:`Apply` to select both the date and time.
```

The email template being used is populated in the :guilabel:`Mail Template` field. The email

> template can be changed by selecting another template form the drop-down menu.

Reviewer 3: ```suggestion
template can be changed by selecting another template from the drop-down menu.
```

.. note::
   Changing the :guilabel:`Mail Template` does **not** change any other part of the email pop-up
   window. Only the email body changes if another email template is selected.

When the email is properly configured, click :guilabel:`Send` to send the interview.

.. _live-session:

Create live session
*******************

To create a live online session where participants can complete the interview form in real-time,
click the :guilabel:`Create Live Session` button in the top-left of the survey form.

The interview survey loads in a new tab. In this tab, several items appear:

- :guilabel:`Attendees`: before anyone is logged in to take the interview survey, a

>   :guilabel:`Waiting for attendees...` box appears on the screen. As attendees login, the box is

Reviewer 3: ```suggestion
  :guilabel:`Waiting for attendees...` box appears on the screen. As attendees log in, the box is
```
Right?

Author: Yes! login and log in always trip me up.

  updated with the number of attendees, and the text reads :guilabel:`Attendees`.
- :guilabel:`QR Code`: the QR code that appears on the screen takes users to the main website, where

>   they can login to take the interview survey.

Reviewer 3: ```suggestion
  they can log in to take the interview survey.
```

- :guilabel:`Start`: once all the attendees have logged in and are ready to begin the interview
  survey, click the :guilabel:`Start` button to begin the interview survey and start the timer.

Once the :guilabel:`Create Live Session` button has been clicked, a :guilabel:`Close Live Session`

> button appears instead, as well as a :guilabel:`See results` button. To end the live session, click

Reviewer 3: ```suggestion
button appears in its place, as well as a :guilabel:`See results` button. To end the live session, click
```
